dev/simpleSamlPhp/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
```

Comment:
Why not add this to the existing compose as another not-always-restart container?